midautumn-2023.html

```diff
@@ -135,12 +135,15 @@
         <style>
             p {
                 color: white;
+                font-size: larger;
+                font-weight: 400;
     
             }
+            
         </style>
+    
         <h1>2023 MID-AUTUMN FESTIVAL Highland Park</h1>
```

Comment:
Can we use different text color for the H1 and H2 here since it is difficult to view the text with the background?